Java/core/src/main/java/com/amazon/randomcutforest/tree/AbstractCompactRandomCutTree.java

```diff
@@ -575,4 +575,8 @@ public T maxSize(int maxSize) {
 
     }
 
+    @Override
+    protected void changeCacheFraction(double fraction) {
```

Comment:
Do we really need the second method here? Instead of adding `changeCacheFraction` to `RandomCutTree` we could just override `setBoundingBoxCacheFraction`:
```
@Override
public void setBoundingBoxCacheFraction(double fraction) {
  super.setBoundingBoxCacheFraction(fraction);
  boxCache.reInitialize(fraction);
}
```

Reply:
Done. Oops :)